tests/js/spec/components/featureBadge.spec.jsx

```diff
@@ -0,0 +1,22 @@
+import {render, screen} from 'sentry-test/reactTestingLibrary';
```

Comment:
can we convert this file to typescript? 🙏

Reply:
Done.